app/controllers/facets_controller.rb

```diff
@@ -95,7 +94,10 @@ def update
 
       redirect_to collection_facets_path(collection.owner, collection), notice: t('collection.facets.collection_facets_updated_successfully')
     else
-      render('collection/facets', :locals => { :@metadata_coverages => collection.metadata_coverages, :@errors => errors })
```

Comment:
global variables now can't be passed thru locals. We should just declare the globals here then